dvc/remote/gdrive.py

```diff
@@ -73,6 +74,14 @@ def __init__(self, repo, config):
         super(RemoteGDrive, self).__init__(repo, config)
         self.no_traverse = False
         self.path_info = self.path_cls(config[Config.SECTION_REMOTE_URL])
+
+        bucket = re.search(
+            "{}://(.*)".format(self.scheme), config[Config.SECTION_REMOTE_URL]
```

Comment:
is it case sensitive/insensitive? does it matter at all?

Reply:
I primarily mean `self.scheme` matching

Reply:
Sounds reasonable, thanks!